src/Shopsys/ShopBundle/Model/Administrator/Administrator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Model\Administrator;
+
+use Doctrine\ORM\Mapping as ORM;
+use Shopsys\FrameworkBundle\Model\Administrator\Administrator as BaseAdministrator;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(
+ *   name="administrators",
+ *   indexes={
+ *     @ORM\Index(columns={"username"})
+ *   }
+ * )
+ */
+class Administrator extends BaseAdministrator
+{
+    public const DEFAULT_ADMIN_ID = 2;
```

Comment:
The name can be confusing – does it mean `(default admin) id` or `default (admin id)`?
But I don't force the change as I don't have any better name.

Reply:
I think it is ok. For your second example I would name it ADMIN_DEFAULT_ID :)